distributed/tests/test_joblib.py

```diff
@@ -140,3 +145,38 @@ def test_joblib_scatter(loop, joblib):
     assert x.count == 1
     assert y.count == 1
     assert z.count == 4
+
+
+@pytest.mark.parametrize('joblib', joblibs)
+def test_nested_backend_context_manager(loop, joblib):
+    if joblib is None or LooseVersion(joblib.__version__) <= "0.11.0":
+        pytest.skip("Joblib >= 0.11.1 required.")
+    Parallel = joblib.Parallel
+    delayed = joblib.delayed
+
+    def get_nested_pids():
+        return Parallel(n_jobs=2)(delayed(os.getpid)() for _ in range(2))
+
+
+    with cluster() as (s, [a, b]):
+        with joblib.parallel_backend('dask.distributed', loop=loop,
+                                     scheduler_host=s['address']) as (ba, _):
```

Comment:
Right, so about the "lets just always use the default client" argument from above, this would become the following:

```python
with Client(s['address'], loop=loop) as client:
    with joblib.parallel_backend('dask') as (ba, _):
        ...
```